### ISSUE TITLE
Setup fails due to missing MICRORAIDEN_VERSION in config.py

### ISSUE THREAD

**Issue by zincoshine** [raiden-network/microraiden#389]
Unable to complete setup due to missing definition of the MICRORAIDEN_VERSION variable in config.py

**Comment by jakjothi** [comment]
Yes, I am also getting same error
```
(pyweb3) aj@ajs-macbook-pro microraiden (master) $ pip install -e .
Obtaining file:///Users/aj/Documents/Blockchain/RaidenNetwork/microraiden/microraiden
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/Users/aj/Documents/Blockchain/RaidenNetwork/microraiden/microraiden/setup.py", line 18, in <module>
        from microraiden.config import MICRORAIDEN_VERSION
    ImportError: cannot import name 'MICRORAIDEN_VERSION'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /Users/aj/Documents/Blockchain/RaidenNetwork/microraiden/microraiden/
```

**Comment by jakjothi** [comment]
Submitted PR #390

**Comment by pcppcp** [comment]
Fixed in PR #390
